pushy/src/main/java/com/eatthepath/pushy/apns/util/LiveActivityApnsPayloadBuilder.java

```diff
@@ -0,0 +1,327 @@
+     * notification. The message in the app's string list may optionally have placeholders, which will be populated by
+     * values from the given {@code alertArguments}.</p>
+     *
+     * @param localizedAlertTitleKey a key to a string in the receiving app's localized string list
+     * @param alertTitleArguments arguments to populate placeholders in the localized alert string; may be {@code null}
+     *
+     * @return a reference to this payload builder
+     *
+     * @see LiveActivityApnsPayloadBuilder#setAlertTitle(String)
+     */
+    public LiveActivityApnsPayloadBuilder setLocalizedAlertTitle(
+            final String localizedAlertTitleKey,
+            final String... alertTitleArguments
+    ) {
+        this.localizedAlertTitleKey = localizedAlertTitleKey;
+        this.localizedAlertTitleArguments = (alertTitleArguments != null && alertTitleArguments.length > 0) ? alertTitleArguments : null;
+
+        return this;
+    }
+
+    /**
+     * <p>Sets the name of the sound file to play when the push notification is received. According to Apple's
+     * documentation, the sound filename should be:</p>
+     *
+     * <blockquote>The name of a sound file in your app’s main bundle or in the {@code Library/Sounds} folder of your
+     * app’s container directory.</blockquote>
+     *
+     * <p>By default, no sound is included in the push notification.</p>
+     *
+     * @param soundFileName the name of the sound file to play, or {@code null} to send no sound
+     *
+     * @return a reference to this payload builder
+     *
+     * @see LiveActivityApnsPayloadBuilder#DEFAULT_SOUND_FILENAME
+     */
+    public LiveActivityApnsPayloadBuilder setAlertSound(final String soundFileName) {
+        this.soundFileName = soundFileName;
+        return this;
+    }
+
+    /**
+     * <p>Adds a custom property inside the content-state payload for Live Activities.</p>
+     *
+     * <p>The precise strategy for serializing the values of custom properties is defined by the specific
+     * {@code ApnsPayloadBuilder} implementation.</p>
+     *
+     * @param key the key of the custom property in the content-state object
+     * @param value the value of the custom property
+     *
+     * @return a reference to this payload builder
+     *
+     * @see <a href="https://developer.apple.com/documentation/activitykit/update-and-end-your-live-activity-with-remote-push-notifications">
+     *      Updating and ending your Live Activity with remote push notifications</a>
+     */
+    public LiveActivityApnsPayloadBuilder addContentStateProperty(final String key, final Object value) {
+        this.contentState.put(key, value);
+        return this;
+    }
+
+    /**
+     * <p>Sets whether the notification payload will be used for updating the Live Activity
+     * or for ending it.</p>
+     *
+     * @param event {@code LiveActivityEvent.UPDATE} to update the Live Activity or
+     * {@code LiveActivityEvent.END} to end it.
```

Comment:
Since this is an enumeration, I think it's okay to be a little less verbose here and let the documentation on `LiveActivityEvent` (that we're definitely going to add, riiiiiight?) explain the options in more detail:

```suggestion
     * @param event the type of update event to send to the destination activity
```